front/lib/models/assistant/conversation.ts

```diff
@@ -244,6 +244,10 @@ export class AgentMessage extends BaseModel<AgentMessage> {
   declare agentConfigurationVersion: number;
 
   declare agentMessageContents?: NonAttribute<AgentMessageContent[]>;
+
+  // Handle to models linked to this agent message
+  declare message?: NonAttribute<Message>;
+  declare feedbacks?: NonAttribute<AgentMessageFeedback[]>;
```

Comment:
also don't see where these are used ?

Reply:
These are used in [this PR](https://github.com/dust-tt/dust/pull/9368) but apparently it's a good practice to change the models in a different PR

they are used to fetch related resources. When you do ~"Include: [message]" you need this nonAttribute, otherwise you can't access the joined field